proto/ibc/tendermint/tendermint.proto

```diff
@@ -68,6 +68,26 @@ message ConsensusState {
   ];
 }
 
+// Evidence is a wrapper over tendermint's DuplicateVoteEvidence
```

Comment:
This is NOT a wrapper over `DuplicateVoteEvidence`. This is really a reimplementation of `ConflictingHeadersEvidence`